client/guidestours/style.scss

```diff
@@ -1,7 +1,33 @@
 .guidestours__step {
 	position: fixed;
-	width: 400px;
-	z-index: z-index( 'root', '.web-preview' );
+	width: 410px;
+	z-index: z-index( 'root', '.guidestours__step' );
+	border: 1px solid lighten( $gray, 10 );
+	box-shadow: 0px 10px 23px 0px rgba( 0, 0, 0, 0.50 );
+	border-radius: 4px;
+	padding-top: 19px;
+	font-size: 14px;
+
+	p {
```

Comment:
I looked through the SCSS / CSS guide and some sample code from the Calypso codebase. I'm not sure what heuristic to follow here: `.button` seems to be a common thing, but for elements like `a` or `p` I've mostly seen them used as elements with no additional classes added. 

What's the rule of thumb to follow here, can you help me understand that better?